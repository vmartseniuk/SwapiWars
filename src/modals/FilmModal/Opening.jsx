/* eslint-disable react/prop-types */
/* eslint-disable react/jsx-one-expression-per-line */
import React, { memo } from "react";
import { makeStyles, Typography } from "@material-ui/core";

const useStyles = makeStyles((theme) => ({
  opening: {
    fontSize: 14,
    lineHeight: "18px",
    letterSpacing: "3px",
    textAlign: "justify",
    padding: "40px 0",
    [theme.breakpoints.up("md")]: {
      fontSize: 18,
      lineHeight: "24px",
    },
  },
}));
const Opening = (props) => {
  const classes = useStyles();
  const { children } = props;
  return <Typography className={classes.opening}>{children}</Typography>;
};

export default memo(Opening);
