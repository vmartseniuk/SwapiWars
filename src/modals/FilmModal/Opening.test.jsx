import { create } from "react-test-renderer";
import Opening from "./Opening";

describe("FilmModal/Opening", () => {
  it("renders correctly", () => {
    const rendered = create(<Opening>text text text</Opening>);
    expect(rendered.toJSON()).toMatchSnapshot();
  });
});
