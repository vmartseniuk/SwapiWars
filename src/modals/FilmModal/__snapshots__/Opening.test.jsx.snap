// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilmModal/Opening renders correctly 1`] = `
<p
  className="MuiTypography-root makeStyles-opening-1 MuiTypography-body1"
>
  text text text
</p>
`;
